src/super_gradients/modules/sampling.py

```diff
@@ -33,6 +33,53 @@ def make_upsample_module(scale_factor: int, upsample_mode: Union[str, UpsampleMo
     return module
 
 
+def make_upsample_module_with_explicit_channels(
```

Comment:
Seems to me like we can extend make_upsample_module functinoality to support exlicit channels, can't we ?

Reply:
I doubt. First, the pixel shuffle implementation differs for this method. The new method uses 1x1 conv + pixel shuffle if channel combination does not match while other method doesn't.
Secondly the new method requires in/out channels and changing the signature in all places where it is used would be quite a task (Since in/output channels number may not be that available). 
Lastly the old method used not only in this repository, so this could be also a BC.